resources/js/components/business-check-in.js

```diff
@@ -1,9 +1,3 @@
-let statusBusiness;
-let statusVisibility;
-let statusBody;
-let statusId          = 0;
-let statusBodyElement = $("#status-body");
-
 let businessCheckInput  = $("#business_check");
```

Comment:
NB: Wo kommt das `$` her? Ich dachte, wir hatten jquery rausgeworfen?

Reply:
jQuery wird noch häufig verwendet und ist noch in den dependencies...